opensoundscape/metrics.py

```diff
@@ -143,11 +143,11 @@ def single_target_metrics(targets, scores, class_names):
     except ValueError:
         metrics_dict["hamming_loss"] = np.nan
     try:
-        metrics_dict["map"] = average_precision_score(targets, preds, average="macro")
+        metrics_dict["map"] = average_precision_score(targets, scores, average="macro")
     except ValueError:
         metrics_dict["map"] = np.nan
     try:
-        metrics_dict["au_roc"] = roc_auc_score(targets, preds, average="macro")
```

Comment:
In a single-target context (except for binary classification), I don't think we should be using average_precision_score. See the note on the SKlearn man page for this function:  https://scikit-learn.org/stable/modules/generated/sklearn.metrics.average_precision_score.html
``` Note: this implementation is restricted to the binary classification task or multilabel classification task. ```

Reply:
Good point, I removed Average Precision score and AU_ROC scores from single_target_metrics, and changed the default evaluation metrics of cnn.CNN.eval() to macro F1 score for single-target models. MAP is still the default evaluation score for multi-target models.